cms/tests/test_urlutils.py

```diff
@@ -19,11 +19,11 @@ def test_is_media_url(self):
             self.assertTrue(urlutils.is_media_request(request))
             request = self.get_request('/no-media/')
             self.assertFalse(urlutils.is_media_request(request))
-        with self.settings(MEDIA_URL='http://testserver2.com/'):
+        with self.settings(MEDIA_URL='http://testserver.com/'):
             request = self.get_request('/')
             self.assertFalse(urlutils.is_media_request(request))
-        with self.settings(MEDIA_URL='http://testserver/media/'):
-            request = self.get_request('/media/')
-            self.assertTrue(urlutils.is_media_request(request))
+        with self.settings(MEDIA_URL='http://testserver.com/media/'):
+            # request = self.get_request('/media/')
```

Comment:
@yakky might also need help here to either rewrite the implementation or change the tests

Reply:
I pushed a fix in 0d2ad18c5

Reply:
awesome